### PR TITLE
Add wording permitting TTML3 and a modular approach.

### LINKED ISSUE #35: Support modularisation for TTML
During the recent face to face meeting, when discussing [support for live contribution](https://github.com/w3c/tt-reqs/issues/3#issuecomment-459298286) the TTWG agreed to pursue a model for an update of TTML, TTML3, in which there will be a modularisation framework, and agreed to add this to the next Charter in such as way as to allow the TTWG to determine the appropriate structure of Rec track documents to work on during its work without requiring a new Charter revision each time a new Rec track document is begun.

Amend the 2019 Charter draft to permit this.

### PR DESCRIPTION
**Pull request by nigelmegitt**
Closes #35.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -125,10 +125,17 @@ <h2>Scope</h2>
           the context of HTML.</p>
         <p>The Group SHOULD:</p>
         <ol class='todo'>
-          <li>Publish a Recommendation for a new <a href="https://www.w3.org/TR/ttml2/">Timed
-              Text Markup Language 2 (TTML2)</a> specification. In the process
-            of producing this new revision, the Group SHOULD:
+          <li>Publish a new <a href="https://www.w3.org/TR/ttml3/">Timed
+              Text Markup Language 3 (TTML3)</a> specification, consisting of 
+              a series of modules where each module and any base document
+              is a Recommendation. In the process
```
Comment (skynavga) [inline index.html]: Change

>consisting of
>a series of modules where each module and any base document
>is a Recommendation.

to read

>a Recommendation that serves as a base document, and which
>refers to one or more functional modules, each to be published
>as a separate Recommendation.

File: index.html [hunk 2]
```diff
@@ -125,10 +125,17 @@ <h2>Scope</h2>
...
+              of producing this new revision, the Group SHOULD:
             <ol type="a">
+              <li>Include a framework defining how each module integrates with
```
Comment (skynavga) [inline index.html]: Change "each module integrates" to "modules integrate".

File: index.html [hunk 3]
```diff
@@ -188,7 +195,12 @@ <h2>Scope</h2>
         </ol>
         <div>
           <h3>Success Criteria</h3>
-          <p>In order to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of each of feature defined in the specification.</p>
+          <p>In order to advance to 
+            <a href="https://www.w3.org/Consortium/Process/#RecsPR"
+            title="Proposed Recommendation">Proposed Recommendation</a>,
+            each specification is expected to have
```
Comment (skynavga) [inline index.html]: Change "specification" to "specification, i.e., either base document or module document,".

File: index.html [hunk 4]
```diff
@@ -208,6 +220,29 @@ <h2> Deliverables </h2>
...
+              </a></dt>
+            <dd>This specification defines a content type that represents timed
+              text media for the purpose of interchange among authoring,
+              distribution and playback systems. Timed text is textual
+              information that is intrinsically or extrinsically associated with
+              timing information. </dd>
+            <dd>
+              <p class="draft-status"><b>Draft state:</b> <i><a href="https://w3c.github.io/ttml3/">Editor's
+                    Draft</a></i></p>
+              <p class="milestone"><b>Expected completion: [Q4-2019]</b> </p>
```
Comment (skynavga) [inline index.html:239]: Change "Q4-2019" to "Q3-2020". The goal is still the former, but we should build in some wiggle room.

File: index.html [hunk 5]
```diff
@@ -223,7 +258,8 @@ <h3> Normative Specifications </h3>
             </dd>
             <dd><b>Adopted Working Draft:</b> Timed
               Text Markup Language 2 (TTML2), <a href="https://www.w3.org/TR/2018/CR-ttml2-20180313/">Candidate Recommendation, 13 March
-              2018.</a>
+              2018</a>.
+
```
Comment (skynavga) [inline index.html:267]: We should update this to refer to a new 2nd edition WD (or ED).

**Review by skynavga — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (nigelmegitt) [reply index.html]: I'm going to tackle this a slightly different way, to refer to Recommendation track technical reports instead, which cover all the specification documents we publish without getting into the weeds of whether each is a specification, a module, a base, or whatever.

File: index.html (hunk 4 above)
Comment (nigelmegitt) [reply index.html:239]: This is only setting an expectation, so there's adequate wriggle room already. I would rather not weaken that expectation right now. Happy to discuss on Thursday's call.

File: index.html (hunk 5 above)
Comment (nigelmegitt) [reply index.html:267]: The deliverables do need updating, but not in this pull request, which is tackling modularisation for TTML3 only. I'll tackle this elsewhere - see https://github.com/w3c/charter-timed-text/pull/42#pullrequestreview-215521606

File: index.html (hunk 2 above)
Comment (nigelmegitt) [reply index.html]: Done.

File: index.html (hunk 1 above)
Comment (nigelmegitt) [reply index.html]: Done.

File: index.html [hunk 6]
```diff
@@ -208,6 +224,29 @@ <h2> Deliverables </h2>
...
           <h3> Normative Specifications </h3>
           <dl class='todo'>
+            <dt class="spec"><a href="https://www.w3.org/TR/ttml3/">Timed Text
+                Markup Language 3 (TTML3)<br>
+              </a></dt>
+            <dd>This specification defines a content type that represents timed
+              text media for the purpose of interchange among authoring,
+              distribution and playback systems. Timed text is textual
+              information that is intrinsically or extrinsically associated with
+              timing information. </dd>
```
Comment (nigelmegitt) [inline index.html:235]: This may consist of one or more documents. Can be not-necessarily TTML3, leave to WG to decide.

File: index.html [hunk 7]
```diff
@@ -125,10 +125,18 @@ <h2>Scope</h2>
           the context of HTML.</p>
         <p>The Group SHOULD:</p>
         <ol class='todo'>
-          <li>Publish a Recommendation for a new <a href="https://www.w3.org/TR/ttml2/">Timed
-              Text Markup Language 2 (TTML2)</a> specification. In the process
-            of producing this new revision, the Group SHOULD:
+          <li>Publish a new <a href="https://www.w3.org/TR/ttml3/">Timed
```
Comment (nigelmegitt) [inline index.html]: Just new versions of TTML as needed.

File: index.html [hunk 8]
```diff
@@ -208,6 +224,29 @@ <h2> Deliverables </h2>
...
+              text media for the purpose of interchange among authoring,
+              distribution and playback systems. Timed text is textual
+              information that is intrinsically or extrinsically associated with
+              timing information. </dd>
+            <dd>
+              <p class="draft-status"><b>Draft state:</b> <i><a href="https://w3c.github.io/ttml3/">Editor's
+                    Draft</a></i></p>
+              <p class="milestone"><b>Expected completion: [Q4-2019]</b> </p>
+            </dd>
+            <dd><b>Adopted Working Draft:</b> Timed
```
Comment (nigelmegitt) [inline index.html:241]: W3 team has tools for generating adopted WDs.

**Review by nigelmegitt — COMMENTED** [review]
Also need to add any Module registry to the list of potential Notes.

File: index.html (hunk 7 above)
Comment (nigelmegitt) [reply index.html]: Done.

File: index.html (hunk 6 above)
Comment (nigelmegitt) [reply index.html:235]: Done.

File: index.html [hunk 9]
```diff
@@ -208,6 +225,29 @@ <h2> Deliverables </h2>
         <div id="normative">
           <h3> Normative Specifications </h3>
           <dl class='todo'>
+            <dt class="spec"><a href="https://www.w3.org/TR/ttml3/">Timed Text
+                Markup Language 3 (TTML3)<br>
+              </a></dt>
+            <dd>This collection of specifications defines a content type that
```
Comment (skynavga) [inline index.html]: Change to "specification". All of the content of this item refers to an Editor's Draft (in the singular) which is not a collection. TTML3 is a single REC track document (=specification) deliverable. If you want, you can a new item after  Timed Text Markup Language 3 (TTML3) that has a title "Timed Text Markup Language 3 (TTML3) Modules", and then list the ones we expect to publish along with the TTML3 specification.

File: index.html [hunk 10]
```diff
@@ -208,6 +225,29 @@ <h2> Deliverables </h2>
...
+              represents timed text media for the purpose of interchange among authoring, distribution and playback systems. 
+              Timed text is textual
+              information that is intrinsically or extrinsically associated with
+              timing information. </dd>
+            <dd>
+              <p class="draft-status"><b>Draft state:</b> <i><a href="https://w3c.github.io/ttml3/">Editor's
+                    Draft</a></i></p>
+              <p class="milestone"><b>Expected completion: [Q4-2019]</b> </p>
```
Comment (skynavga) [inline index.html:239]: I'm not comfortable saying Q4 is "expected". While that may be our goal, I would be more comfortable saying "Q3-2020" for expected.

**Review by skynavga — CHANGES_REQUESTED** [review]

File: index.html (hunk 10 above)
Comment (nigelmegitt) [reply index.html:239]: @skynavga to do that would be a divergence from the working mode we agreed to adopt, in which features that we could not get through the process during this year would be deferred. If we do not expect to publish until Q3 2020 then I question whether we should even start. There was a lot of pressure caused by how long TTML2 took, and I think there was agreement to do things differently next time.

**Comment by nigelmegitt** [comment]
@skynavga I've attempted to address your review comments - if we are going to change the expected publication date of TTML3 we need a WG discussion about that because I think the group has clear expectations about that already.

File: index.html (hunk 9 above)
Comment (nigelmegitt) [reply index.html]: Done.

**Review by skynavga — APPROVED** [review]

File: index.html (hunk 10 above)
Comment (skynavga) [reply index.html:239]: @nigelmegitt now that you have updated the deliverable info for TTML3 to one spec, it is possible to meet that schedule; however, I have doubts that all the issues/requirements that we have delegated to modules will complete

**Pull request merged**